docs/cloud/introduction/support.md

```diff
@@ -47,3 +88,12 @@ To request assistance from Temporal Support, create a ticket in Zendesk:
      If you're not sure, select **Normal**.
    - Select **Page** only if you are experiencing a complete service outage and urgently need contact with an on-call support person.
 1. At the bottom of the form, choose **Submit**.
+
+## What type of training is available? {#training}
+
+Temporal offers coursework and a variety of hands-on tutorials to get you started and learn more advanced Temporal concepts.
+
+- [Get started with Temporal](https://learn.temporal.io/getting_started): Start your journey with Temporal with this guide that helps you set up your development environment, run an existing Temporal app, and then build your first app from scratch using our SDKs.
```

Comment:
Do we purposefully have the same excerpt in docs-src/cloud/support-training.md?

Reply:
This is the source where all other files build to.

Support-training is the invidual page that makes this one up.